Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM gcr.io/swift-tensorflow/base-deps-cuda10.1-cudnn7-ubuntu18.04
 
 # Allows the caller to specify the toolchain to use.
-ARG swift_tf_url=https://storage.googleapis.com/swift-tensorflow-artifacts/nightlies/latest/swift-tensorflow-DEVELOPMENT-cuda10.1-cudnn7-ubuntu18.04.tar.gz
+ARG swift_tf_url=https://storage.googleapis.com/swift-tensorflow-artifacts/custom-build/latest/swift-tensorflow-DEVELOPMENT-notf-ubuntu18.04.tar.gz
```

Comment:
You should use the "notf" toolchain at `https://storage.googleapis.com/swift-tensorflow-artifacts/nightlies/latest/swift-tensorflow-DEVELOPMENT-notf-ubuntu18.04.tar.gz`. It's not there yet, but the first automatically triggered build is almost finished and it should appear there any minute now...

Reply:
Ok, it's there now!